source/lmp/pair_deepmd.cpp

```diff
@@ -563,60 +200,26 @@ void PairDeepMD::compute(int eflag, int vflag) {
         commdata_->nswap, commdata_->sendnum, commdata_->recvnum,
         commdata_->firstrecv, commdata_->sendlist, commdata_->sendproc,
         commdata_->recvproc, &world);
-    lmp_list.set_mask(NEIGHMASK);
```

Comment:
#4269 should not be reverted, and you may also need to apply it to your new code.